ddlite.py

```diff
@@ -78,6 +90,8 @@ class candidate_internal(object):
   See entity_internal and relation_internal for examples
   """
   def __init__(self, all_idxs, labels, sent, xt):
+    self.uid = "{}::{}::{}::{}".format(sent.doc_name, sent.sent_id,
```

Comment:
@henryre what's the difference between doc_name and doc_id? Is it possible to have multiple docs in the same file? Do you think using `doc_id` here would be safer?